src/promo/DonationDialog.jsx

```diff
@@ -27,9 +28,9 @@ const styles = theme => ({
   paper: {
     // TODO: revisit palette colours with what we learned building
     // this component.
-    backgroundColor: theme.palette.core && theme.palette.core.main,
+    backgroundColor: corePalette[600],
     overflow: 'visible', // So we can have the avatar poking out the top
-    color: theme.palette.primary.contrastText,
+    color: '#fff',
```

Comment:
And here too. 

color: neutralPalette[0]

Reply:
Aaaand done.